imap_processing/glows/l1/glows_l1a_data.py

```diff
@@ -200,14 +242,27 @@ def __init__(self, level0: HistogramL0):
         #                      f"[{self.l0.SPINS}]")
 
         # Create time tuples based on second and subsecond pairs
-        self.imap_start_time = TimeTuple(self.l0.SEC, self.l0.SUBSEC)
-        self.imap_time_offset = TimeTuple(self.l0.OFFSETSEC, self.l0.OFFSETSUBSEC)
-        self.glows_start_time = TimeTuple(self.l0.GLXSEC, self.l0.GLXSUBSEC)
-        self.glows_time_offset = TimeTuple(self.l0.GLXOFFSEC, self.l0.GLXOFFSUBSEC)
+        self.imap_start_time = TimeTuple(l0.SEC, l0.SUBSEC)
+        self.imap_time_offset = TimeTuple(l0.OFFSETSEC, l0.OFFSETSUBSEC)
+        self.glows_start_time = TimeTuple(l0.GLXSEC, l0.GLXSUBSEC)
+        self.glows_time_offset = TimeTuple(l0.GLXOFFSEC, l0.GLXOFFSUBSEC)
+
+        # In L1a, these are left as unit encoded values.
+        self.number_of_spins_per_block = l0.SPINS
+        self.number_of_bins_per_histogram = l0.NBINS
+        self.number_of_events = l0.EVENTS
+        self.filter_temperature_average = l0.TEMPAVG
+        self.filter_temperature_variance = l0.TEMPVAR
+        self.hv_voltage_average = l0.HVAVG
+        self.hv_voltage_variance = l0.HVVAR
+        self.spin_period_average = l0.SPAVG
+        self.spin_period_variance = l0.SPAVG
```

Comment:
Duplicate entry, is there a VAR here too?
```suggestion
        self.spin_period_average = l0.SPAVG
        self.spin_period_variance = l0.SPVAR
```